indico/modules/events/client/js/components/AutoLinkerConfig.jsx

```diff
@@ -0,0 +1,135 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import autoLinkerConfigUrl from 'indico-url:events.autolinker_config';
+
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import {Button, Form, List, Message, Segment} from 'semantic-ui-react';
+
+import {handleSubmissionError} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+import {handleAxiosError, indicoAxios} from 'indico/utils/axios';
+
+import './AutoLinkerConfig.module.scss';
```

Comment:
move this one after the proptypes import below and eslint will be able to properly sort the import (any import with side-effects like those css imports creates a new group for sorting, since it cannot know whether it's safe to move imports between the two sides of such an import)